### ISSUE TITLE
team black

### ISSUE THREAD

**Issue by esraneufeld** [ITISFoundation/osparc-simcore#695]
prepare proposal for UX feedback backlog and process
setup infrastructure

**Comment by odeimaiz** [comment]
Following workflow implementation will be provided by the development team:
1. User clicks "Open issue" in the user's menu.
2. A window will pop up showing either an iframe pointing to github [new issue page](https://github.com/ITISFoundation/osparc-simcore/issues/new?template=bug_report.md) (some research needed since github does not allow their pages to be embedded into iframes) or a form expecting title and description. Together with this information, some environment information will be collected (platform version, GUI version, 3rd party libs versions). In following iterations more information, such as screenshots, steps to reproduce..., will be added to the issue.
3. The issue gets added to the [list of issues](https://github.com/ITISFoundation/osparc-simcore/issues) with the "tester bug" label, so that they can be easily filtered.
4. In order to provide traceability, once it is added, a ticket (link to the issue itself) will be displayed. From there, the issue creator will be able to subscribe/unsubscribe to its updates (subscribed by default).
5. After internal/external discussions these issues will get a priority and a milestone.

**Comment by odeimaiz** [comment]
### Concepts
**Issue**: Item that collects user feedback, reports software bugs or organizes feature requests.
**Backlog**: Panel where all issues are displayed. Issues can also be tagged/label, therefore, these can be easily filtered in the main panel by labels.
**Sprint**: Period of time during which a certain issue is tackled and if completed, once the period is finished, reviewed.
**Milestone**: Deadline by when the issue should be fixed/implemented/delivered.

### Workflow
1. User clicks "Open issue" in the frontend's user menu.
2. A window will pop up where the user/tester will provide some information about the feedback, bug or feature request.
3. The issue will be added to a dedicated backlog with a specific label.
4. After internal/external discussions these issues will get a priority.
5. Before starting the a new sprint cycle, what issues will be tackled is discussed and they get a milestone.
6. Once an issue is completed and reviewed, it is closed and the issue creator gets a notification.
